swarm/swarm.go

```diff
@@ -216,7 +220,7 @@ func (self *Swarm) Stop() error {
 	if self.lstore != nil {
 		self.lstore.DbStore.Close()
 	}
-
+	self.sfs.Stop()
```

Comment:
can something foul happen if we do not unmount active mounts. E.g. KILLSIG?
SIGINT and recently also SIGTERM are caught gracefully